### DISCUSSION TITLE
Could not find module 'SwiftUICharts' for target 'x86_64-apple-ios-simulator'; found: arm64, arm64-apple-ios-simulator

### DISCUSSION THREAD

**Discussion by growly** [AppPear/ChartView#216] [Q&A]
Has anyone else run into this problem when trying to include the SwiftUICharts package? I'm building on an M1 mac mini. I use CocoaPods for Firebase libraries.

I've Google'd the problem to death, but no cigar. I'm fairly certain there's some problem with my configuration, but none of the suggestions I've seen have helped.

The failing target:
- has 'Build Active Architecture Only' set to NO for Automation, Release and YES for Debug
- has 'Excluded Architectures' set to 'arm64' for all
- targets iOS v14.1

I've tried:
- adding 'VALID_ARCHS = arm64 x86_64 [...]' as a user-defined setting
- removing all 'Excluded Architectures'
- set 'Build Active Architecture Only' to YES
- set 'Build Active Architecture Only' to NO
- adding 'x86_64', 'x86_64-apple-ios-simulator', etc to 'Architectures'

I do not understand the xcode build system, so perhaps a good reference for that would be helpful too.

Thanks in advance!

**Comment by growly** [comment]
I created a new Xcode project and then had no problem integrating SwiftUICharts. The problem is probably because of 'EXCLUDED_ARCHS', which I have to allow Firebase pods to build.

Ugh.

**Comment by deadsoftie** [comment]
Check whether the naming of your package is correct. If you want to use the beta make sure the Package Dependency rule is set to exact version and that you type in exactly 2.0.0-beta.2 otherwise the error will pop up. If you want to use a stable version set  Package Dependency rule to Up to next major version and then type version 1.5.4. This is should solve your problem.
